server/graphql/v2/interface/Account.js

```diff
@@ -185,6 +188,13 @@ const accountFieldsDefinition = () => ({
       },
     },
   },
+  memberInvitations: {
+    description: 'Get pending member invitations for this account',
+    type: new GraphQLList(MemberInvitation),
```

Comment:
What's the thought process to have a simple `GraphQLList` instead of a `*Collection` structure?

Reply:
Ok, to match the behavior of the existing `memberInvitations` top level query.

Reply:
@znarf exactly. I also think that, since this list should be pretty reduced we won't need to paginate it.
I'm OK with updating this if we decide every list should be collections.